scripts/fetch_tools.sh

```diff
@@ -216,6 +216,17 @@ gettool_awscli() {
     popd &>/dev/null
 }
 
+gettool_oc() {
+    local ver="4.16.3"
+    pushd "${WORK_DIR}" &>/dev/null
+
+    curl -s "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/${ver}/openshift-client-linux-${ver}.tar.gz" -o "oc-${ver}-linux.tar.gz"
```

Comment:
You'll need to dynamically set the arch here since MicroShift supports both x86_64 and arm64

Reply:
Also, probably want to use the `-f` or `--fail-with-body` curl flags, otherwise http codes >400 will still be considered successes.